src/themes/default/components/core/blocks/MyAccount/MyOrder.vue

```diff
@@ -63,35 +66,38 @@
               <td class="fs-medium lh25" :data-th="$t('Subtotal')">
                 {{ item.row_total_incl_tax | price }}
               </td>
+              <td class="fs-medium lh25">
+                <img :src="getProductThumbnail(item.sku)">
```

Comment:
i would be nice to use ProductImage component here, to show placeholders until proper product image is loaded